lang/main-te.json

```diff
@@ -44,7 +44,7 @@
         "audioOnly": "తక్కువ బ్యాండ్‌విడ్త్"
     },
     "calendarSync": {
-        "addMeetingURL": "Add a meeting link",
+        "addMeetingURL": "సమావేశ లంకెను చేర్చు",
```

Comment:
There seems to be a typo here.

Reply:
లంకె is Telugu of లింకు. If that was your comment about.

Reply:
Well, I learnt a new telugu word today. Thanks for clarifying.